### ISSUE TITLE
Make cloudscribe page headings <h1>

### ISSUE THREAD

**Issue by StewartBellamy** [cloudscribe/cloudscribe#933]
Currently cs page headings are h2 and we're failing accessibility tests.
Privacy, registration agreement.

**Comment by StewartBellamy** [comment]
Resolved in cloudscribe.Core.Bootstrap5@6.0.22

**Comment by StewartBellamy** [comment]
Login, password, access denied etc.

Resolved in cloudscribe.Core.Bootstrap5@6.0.23
